B2B 2.4.2 additions (#840)
magento/merchdocs: src/marketing/price-rules-cart-create.md

   - Total Items Quantity
   - Total Weight

 {:.bs-callout-warning}
 For Purchase Orders only: When a Cart Price Rule is set based on Payment Method, and the payment method is changed after the purchase order is approved, the discount remains applied to the total.

> 

Reviewer: It should be after the purchase order is `created`, not `approved`.

Author: Good catch.

>  For purchase orders only: When a Cart Price Rule is set based on a specific Payment Method(s), the discount is applied to the total when a purchase order is created. If, after the purchase order is created, the payment method is changed to something not covered by the Cart Price rule, the discount remains applied to the total.

Reviewer 2: The use of `(s)` tends to make things awkward. I would suggest:

```suggestion
 For purchase orders only: When a Cart Price Rule is set based on one or more specific Payment Methods, the discount is applied to the total when a purchase order is created. If, after the purchase order is created, the payment method is changed to one that is not covered by the Cart Price rule, the discount remains applied to the total.
```
